apps/frontend/types/object.types.ts

```diff
@@ -331,3 +331,17 @@ export type RegistryCardData = AutomationResponse & {
   verifiedTooltip?: string;
   filterIconDictionary?: AutomationFilterIconDictionary;
 };
+
+export type GithubRepository = {
```

Comment:
we have this type already defined in `apps/backend/src/services/SourceControl.ts` 
l think it was lost during some merge but I created @be-types for this. I think Dmytro already moved it back so it could be used